Repository/PackageRepositoryCreateParameters.cs

```diff
@@ -0,0 +1,83 @@
+﻿//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+namespace Microsoft.PackageManagement.NuGetProvider
+{
+    using System;
+
+    /// <summary>
+    /// Parameters to create IPackageRepository
+    /// </summary>
+    public class PackageRepositoryCreateParameters
```

Comment:
is this similar to PackageItem?

Reply:
This is not a representation of a package.

---
In reply to: [154461721](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154461721) [](ancestors = 154461721)